operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -853,7 +853,7 @@ def customizeNetworkAccessPoint(self, nap, listen_address):
     # FWIW there's theoretically no need to 'add' or 'replace' when empty
     #   since the runtime default is the server listen-address.
     nap_name=nap.getName()
-    if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0:
+    if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0 and not (nap_name.startswith('istio-')) and not (nap_name == 'T3Channel'):
```

Comment:
check the logic here - does this break the non-istio case?

Reply:
fixed